parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -284,7 +284,7 @@ impl pallet_timestamp::Config for Runtime {
 }
 
 parameter_types! {
-    pub const TransactionByteFee: Balance = 0;  // TODO: this is 0 so that we can do runtime upgrade without fees. Restore value afterwards!
+    pub const TransactionByteFee: Balance = 1;
```

Comment:
Shouldn't we set it to a more realistic value? I'm not sure how to arrive at such a value though.,

Reply:
This is what was set previously, would need to check the effects upstream.

Reply:
It is just an additional cost per byte of the encoded call - basically to pay for decoding them